sdk/client/src/blocking.rs

```diff
@@ -233,6 +238,54 @@ impl BlockingClient {
         self.send(MethodRequest::get_events_with_proofs(key, start_seq, limit))
     }
 
+    /// Return the events of type `T` that have been emitted to `event_key` since `start_seq`, with a max of `limit`
+    /// results
+    /// Returns an empty vector if there are no such events
+    pub fn get_deserialized_events<T: MoveResource + DeserializeOwned>(
```

Comment:
I wonder what is not exposed through existing get_events API, or we should return event bytes in the get_events response.
as it uses verifying api, maybe get_unverified_events is better naming pattern.

Reply:
Yeah I wonder if we could add an API to query some of this raw data as right now its fairly difficult

Reply:
I think adding this to json-rpc for both the event and the account API's would make sense.

Reply:
Is there any unknown event? Or is it prepare for future to have unknown events? Given your implementation, it is more like we need output event data bytes for all events, but your link is pointing to "unknown" event data

Reply:
assuming we should have right struct to decode everything json-rpc event view knows, I think add bytes to unknown event data is right, so here we go: https://github.com/diem/diem/pull/8241

Reply:
It is to prepare for:
- A future with custom module publishing where we can't put every possible event type in json-rpc
- A future where adding a new event type in Diem requires less boilerplate (e.g., we can add a new event type and allow clients to serialize it without modifying json-rpc).

Reply:
That's effectively what the API Sam is adding is doing though right? Allowing you to query raw event data bytes (of course it requires that you to attempt to deserialize an event type instead of just giving you bytes though.